packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1055,35 +1502,25 @@ class RoundedRectSliderTrackShape extends SliderTrackShape with BaseSliderTrackS
 ///
 /// See also:
 ///
-///  * [Slider], for the component that is meant to display this shape.
+///  * [RangeSlider], for the component that is meant to display this shape.
 ///  * [SliderThemeData], where an instance of this class is set to inform the
 ///    slider of the visual details of the its track.
-///  * [SliderTrackShape], which is the base component for creating other
+///  * [RangeSliderTrackShape], which is the base component for creating other
 ///     custom track shapes.
-///  * [RoundedRectSliderTrackShape], for a similar track with rounded edges.
-class RectangularSliderTrackShape extends SliderTrackShape {
-  /// Create a slider track that draws 2 rectangles.
-  const RectangularSliderTrackShape({ this.disabledThumbGapWidth = 2.0 });
-
-  /// Horizontal spacing, or gap, between the disabled thumb and the track.
-  ///
-  /// This is only used when the slider is disabled. There is no gap around
-  /// the thumb and any part of the track when the slider is enabled. The
-  /// Material spec defaults this gap width 2, which is half of the disabled
-  /// thumb radius.
-  final double disabledThumbGapWidth;
+///  * [RoundedRectRangeSliderTrackShape], for a similar track with rounded edges.
+class RectangularRangeSliderTrackShape extends RangeSliderTrackShape {
+  /// Abstract const constructor. This constructor enables subclasses to provide
```

Comment:
I don't think this is the right documentation for the constructor since this isn't an abstract class. Might be a copy-paste leftover.

Reply:
Fixed